README.md

```diff
@@ -96,11 +96,20 @@ Of course you want to get a list of notifications per user. Here are some exampl
     // getting a list of all notifications of the user with id 2
     $this->Notifier->getNotifications(2);
     
-    // getting a list of all unread notifications
-    $this->Notifier->allNotificationList(2, true);
+    // getting a list of all unread notifications for user with ID 2
```

Comment:
I documented with 'id', here you did it with 'ID' and below you typed 'Id'. What will it be? I suggest 'id'.